### PR TITLE
many: mv keys to ubuntu-boot, move model file, rename keyring prefix for secboot

### PR DESCRIPTION
**Pull request by anonymouse64**
* Move sealed-key for ubuntu-data to ubuntu-boot
* Move model file to devicedir
* Rename keyring prefix

### CODE REVIEW

**Review by pedronis — APPROVED** [review]
thank you

File: tests/core/basic20/task.yaml [hunk 1]
```diff
@@ -33,6 +33,9 @@ execute: |
     test -e /boot/grub/"$kernel_name"*/kernel.efi
     test -e /boot/grub/kernel.efi
 
+    echo "Ensure that model was written to ubuntu-boot"
+    test -e /run/mnt/ubuntu-boot/device/model
```
Comment (mvo5) [inline tests/core/basic20/task.yaml:37]: Thanks for adding this test!

**Review by mvo5 — APPROVED** [review]
Thank you

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**